src/pages/dashboardGroupEdit/CloneGroupPage.tsx

```diff
@@ -0,0 +1,78 @@
+import { Grid } from '@mui/material';
+import { TitleBox } from '@pagopa/selfcare-common-frontend';
+import { resolvePathVariables } from '@pagopa/selfcare-common-frontend/utils/routes-utils';
+import { useHistory } from 'react-router-dom';
+import ProductNavigationBar from '../../components/ProductNavigationBar';
+import withGroupDetail, { withGroupDetailProps } from '../../decorators/withGroupDetail';
+import { PartyGroupOnEdit } from '../../model/PartyGroup';
+import { Product } from '../../model/Product';
+import { DASHBOARD_ROUTES } from '../../routes';
+import GroupForm from './components/GroupForm';
+
+type Props = {
+  activeProducts: Array<Product>;
+  partyGroupCloneId: string;
+} & withGroupDetailProps;
+
+function CloneGroupPage({
+  party,
+  activeProducts,
+  productsMap,
+  partyGroup,
+  partyGroupCloneId,
+}: Props) {
+  const history = useHistory();
+
+  const paths = [
+    {
+      description: 'Gruppi',
+      onClick: () =>
+        history.push(
+          resolvePathVariables(DASHBOARD_ROUTES.PARTY_GROUPS.subRoutes.MAIN.path, {
+            institutionId: party.institutionId,
+            groupId: partyGroup.id,
+          })
+        ),
+    },
+    {
+      description: 'Duplica gruppo',
+    },
+  ];
+
+  return (
+    <Grid
+      container
+      alignItems={'center'}
+      px={2}
+      mt={10}
+      sx={{ width: '985px', backgroundColor: 'transparent !important' }}
+    >
+      <Grid item xs={12} mb={3}>
+        <ProductNavigationBar paths={paths} />
+      </Grid>
+      <Grid item xs={12} mb={9}>
+        <TitleBox title="Duplica gruppo" subTitle={`Duplica il gruppo e modifica i dati`} />
+      </Grid>
+      <Grid item xs={12}>
+        <GroupForm
+          party={party}
+          products={activeProducts}
+          productsMap={productsMap}
+          initialFormData={
+            {
+              id: partyGroupCloneId,
```

Comment:
This should not be filled! in this way you are providing the same id of the cloned group to the new group, which will brought to the update of the previous group!